src/js/libgeo/Prover.js

```diff
@@ -45,6 +45,9 @@ function applyIncidence(a, b) {
 
 function incidentPL(p, l) {
     return {
+        getInvolved: function() {
+            return [p, l];
+        },
```

Comment:
Get involved and receive a pair of geometric objects for free! ;-)

Thanks for the annotations Martin, i'll look into it.

Am 01.04.2016 um 00:12 schrieb Martin von Gagern:
>
> In src/js/libgeo/Prover.js
> <https://github.com/CindyJS/CindyJS/pull/299#discussion_r58136437>:
>
> > @@ -45,6 +45,9 @@ function applyIncidence(a, b) {
> >  
> >  function incidentPL(p, l) {
> >      return {
> > +        getInvolved: function() {
> > +            return [p, l];
> > +        },
>
> If I didn't know better, I'd read the name here as an invitation for
> possible future contributors: /Get involved with the CindyJS project!/
> :smile:
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/CindyJS/CindyJS/pull/299/files/13207c592c99c82ab73a5bdda83d9586b1a9e53d#r58136437>
>

-- 
Michael Strobel, M.Sc.

Technische Universität München -- Zentrum Mathematik
Lehrstuhl für Geometrie und Visualisierung (M10)
Boltzmannstr. 3, 85747 Garching bei München 
https://www-m10.ma.tum.de/bin/view/Lehrstuhl/MichaelStrobel